nuxt.config.ts

```diff
@@ -1,73 +1,79 @@
 export default defineNuxtConfig({
-  routeRules: {
-      '/api/search': {
-          prerender: true
-      }
-  },
+    routeRules: {
+        '/api/search': {
+            prerender: true
+        }
+    },
 
-  devtools: { enabled: true },
+    devtools: { enabled: true },
 
-  modules: [
-      '@nuxtjs/tailwindcss',
-      '@nuxt/content',
-      '@nuxt/image',
-      '@stefanobartoletti/nuxt-social-share'
-  ],
+    // Easy access to enabling sourcemaps for debugging
+    sourcemap: {
```

Comment:
is it the only difference ?
Is it possible to keep the formatting of the previous file. It helps to understand the difference between the two versions

Reply:
Yes, that is the only difference. 

The diff would have been prior to 104da4267c2285f5e0d2caaea97f9824f65fd6bd:

```diff
diff --git a/nuxt.config.ts b/nuxt.config.ts
index a03ccca..c358244 100644
--- a/nuxt.config.ts
+++ b/nuxt.config.ts
@@ -6,6 +6,13 @@ export default defineNuxtConfig({
     },
 
     devtools: { enabled: true },
+
+    // Easy access to enabling sourcemaps for debugging
+    sourcemap: {
+        // server: true,
+        // client: true
+    },
+
     modules: [
         '@nuxtjs/tailwindcss',
         '@nuxt/content',
```
[GitHub compare link](https://github.com/bloggrify/bloggrify/compare/b34e47d485d732cfaf40b8aa2b31c668880a5069...e660299faeb19c0c74d6f710232f60d1f99cca02)

As the `main` branch was updated with rearranging / formatting in 104da4267c2285f5e0d2caaea97f9824f65fd6bd it may not look very clean. The fork was synced to ensure it would not override the upstream `main` branch.